plugins/arDominionB5Plugin/js/fullWidthTreeView.js

```diff
@@ -368,6 +372,42 @@
         this.showAlert(moveResponse.success, "alert-info");
       }
     };
+
+    hoverNodeListener = (e, data) => {
+      var parent = data.node.parent;
+      var parentNode = this.$fwTreeView.jstree("get_json", parent);
```

Comment:
Not a big deal in this case, but we should start using `let` instead of `var` more often.

Reply:
Updated! And yes, all in favour of going for `const` and `let` instead of `var` moving forward.